src/components/Header/Header.js

```diff
@@ -1,30 +1,34 @@
-import React from 'react'
-import { IndexLink, Link } from 'react-router'
-import classes from './Header.scss'
-
-export const Header = () => (
-  <div>
-    <h1>Project Homeless Connect</h1>
-    <IndexLink to='/' activeClassName={classes.activeRoute}>
-      Home
-    </IndexLink>
-    {' · '}
-    <Link to='/counter' activeClassName={classes.activeRoute}>
-      Counter [for reference]
-    </Link>
-    {' · '}
-    <Link to='/check-in' activeClassName={classes.activeRoute}>
-      Check In
-    </Link>
-    {' · '}
-    <Link to='/services' activeClassName={classes.activeRoute}>
-      Services
-    </Link>
-    {' · '}
-    <Link to='/zen' activeClassName={classes.activeRoute}>
-      Zen
-    </Link>
-  </div>
-)
-
-export default Header
+import React from 'react'
+import { IndexLink, Link } from 'react-router'
+import classes from './Header.scss'
+
+export const Header = () => (
+  <div>
+    <h1>Project Homeless Connect</h1>
+    <IndexLink to='/' activeClassName={classes.activeRoute}>
+      Home
+    </IndexLink>
+    {' · '}
+    <Link to='/counter' activeClassName={classes.activeRoute}>
```

Comment:
does git have a really crappy diffing algo or something?  how can these be so different lol